.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -0,0 +1,82 @@
+name: "Bug Report"
+description: "Create a report to help us improve"
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Thank you for reporting a bug!** Please fill out the following sections as completely as possible.
+
+  - type: textarea
+    id: bug_description
+    attributes:
+      label: "Bug Description"
+      description: "Provide a clear and concise description of the issue, including steps to reproduce it."
+      placeholder: "What happened? What did you expect to happen?"
+    validations:
+      required: true
+
+  - type: input
+    id: os_version
+    attributes:
+      label: "OS Version"
+      description: "Example: Ubuntu 22.04, macOS 14.0, Windows 11"
+    validations:
+      required: true
+
+  - type: input
+    id: app_version
+    attributes:
+      label: "App Version"
+      description: "Find this in the About section or GitHub releases."
+    validations:
+      required: true
+
+  - type: dropdown
+    id: mode
+    attributes:
+      label: "Mode"
```

```suggestion
      label: "RLN node connection type"
```
please rename this also in the app (now it says "select wallet connection type" which is imprecise)